src/importFreshWrapper.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+//
+// this is one solution to make importFresh spy'able
+// see: https://github.com/sinonjs/sinon/issues/664
+//
+
+const importFresh = require('import-fresh');
+
+module.exports = { importFresh };
```

Comment:
With Jest we can mock modules with [`jest.mock`](https://jestjs.io/docs/en/jest-object#jestmockmodulename-factory-options), so I don't think this wrapper is necessary.

Reply:
Ugh - I'm sorry but I'm having trouble figuring out how to organize the mock code and would appreciate some direction.  I tried adding

```js
const mockedRequireFresh = jest.fn(() => ({ foundJs: true }));
jest.mock('import-fresh', mockedRequireFresh);

const cosmiconfig = require('../src');
```

but then something happens to the mock by the time the test runs where it returns undefined.  From what I read `jest.restoreAllMocks` shouldn't be at fault and that proved true when I commented that line out.

I haven't used jest before.  It has a large api surface and seems to be doing some magical things (e.g. its own `require` took me a bit to figure out).